graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -186,36 +230,35 @@ class RenameSchemaTypesVisitor(Visitor):
         }
     )
 
-    def __init__(self, renamings, query_type, scalar_types):
+    def __init__(
+        self, renamings: Mapping[str, str], query_type: str, scalar_types: Set[str]
```

Comment:
```suggestion
        self, renamings: Mapping[str, str], query_type: str, scalar_types: AbstractSet[str]
```
https://docs.python.org/3/library/typing.html#typing.AbstractSet will allow both `set()` and `frozenset()`. This variant only allows `set()` specifically.